src/pages/Error404/Error404.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import styles from "./styles.css";
+
+import GalaxyBg from "../../assets/images/GalaxyBg.png";
+
+import Error404Text from "../../components/Error404Text/Error404Text";
+import RedirectAlert from "../../components/RedirectAlert/RedirectAlert";
+
+const Error404: React.FC = () => (
```

Comment:
Not sure of naming
things should by described by words rather than `HTTP response code`
`NotFound` is more self explaining
also it's complete view so adding `view` is advisable

as NotFound is only my sugestion, naming this thing `view` is indeed required to solve this comment